test/Chainweb/Test/Pact/SPV.hs

```diff
@@ -262,7 +262,7 @@ roundtrip'
     -> IO (CutOutputs, CutOutputs)
 roundtrip' v sid0 tid0 burn create step = withTestBlockDb v $ \bdb -> do
   tg <- newMVar mempty
-  withWebPactExecutionService v bdb (chainToMPA' tg) $ \pact -> do
+  withWebPactExecutionService v bdb (chainToMPA' tg) freeGasModel $ \pact -> do
```

Comment:
How much test coverage of gas computation code do we loose by using `freeGasModel` in so many places in the tests?

Reply:
We don't lose any test coverage, we were already using `freeGasmodel`.

We can make a case for refactoring the tests that do this to use the proper gas model later on.